src/shared/components/pipeline-run-logs/logs/Logs.tsx

```
@@ -1,161 +1,232 @@
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
-import { Alert } from '@patternfly/react-core';
 import { Base64 } from 'js-base64';
-import { useNamespace } from '~/shared/providers/Namespace';
+import { FLAGS } from '~/feature-flags/flags';
+import { useIsOnFeatureFlag } from '~/feature-flags/hooks';
+import { FeatureFlagsStore } from '~/feature-flags/store';
```

ESLint

[error] 'FeatureFlagsStore' is defined but never used. Allowed unused vars must match /React/u.

(@typescript-eslint/no-unused-vars)